README.asciidoc

```diff
@@ -369,6 +369,19 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Scripting]]
+Hints on scripted usage
+-----------------------
+If you want to run image-bootstrap from a shell script and capture its output
+to STDOUT/STDERR in a log file, be sure to run image-bootstrap in unbuffered
```

Comment:
Would you mind going lowercase on "STDOUT/STDERR", with code-backticks maybe, e.g. `stdout`/`stderr`? It looks weird uppercase and Wikipedia has is lowercase as well: https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout). Or text version "standard output/error"? I think I like `stdout`/`stderr` best. Two more occurrences below.

Reply:
ack